frontend/packages/client/src/components/Community/CommunityEditorProfile.js

```diff
@@ -164,7 +193,7 @@ function CommunityEditorProfile({
                 }}
               />
             )}
-            {!(isUpdating && image.file) ? (
```

Comment:
Not sure about this change, I know it's kind of tricky but I didn't want to show the message 'Uploading' on top of the image when the user was just updating the community name and not uploading a new image, since the variable isUpdating comes form the hook I didn't have a better way to check what kind of update was done. If we change this, then uploading will show up for any text update on the community. Also if you didn't add any new image and want to keep it with the bloquies avatar it will still show "Uploading..." 
<img width="406" alt="Screen Shot 2022-07-21 at 11 31 50" src="https://user-images.githubusercontent.com/7526740/180241226-d6cb8b52-01e7-48de-9625-994b0419859f.png">

Reply:
its not the prettiest but ill add `isUpdatingImage` and `isUpdatingBanner` to distinguish this logic